include/hyro/DigitalConverterComponent.h

```diff
@@ -0,0 +1,96 @@
+/**
+ * @file DigitalConverterComponent.h
+ * @author Luciana Moreno Borges (lumoreno.borges@gmail.com)
+ * @brief Digital Converter component 
+ * @version 0.1
+ * @date 2019-01-25
+ * 
+ * @copyright Copyright (c) 2019
+ * 
+ */
+#pragma once
+
+#ifndef HYRO_DIGITALCONVERTERCOMPONENT_H
+#define HYRO_DIGITALCONVERTERCOMPONENT_H
+
+#include <signal_generator_components_export.h>
+
+#include <hyro/core/Component.h>
+#include <hyro/utils/ISpinner.h>
+
+#include <hyro/commands/CommandHandler.h>
+
+#include <hyro/msgs/Signal.h>
+#include <hyro/Thresholding.h>
+
+namespace hyro
```

Comment:
We use indentation size of 2 spaces. Please configure your editor to use 2 spaces.

Usually we do not indent the class with respect to the namespace. See the example below:

```c++
namespace hyro
{
class DigitalConverterComponent : public Component
{
public:
    void init();

private:
   int m_value;

}; // class DigitalConverterComponent
} // namespace hyro

Reply:
This is valid for all the files in the project, e.g. `SignalGenerator.h`, `Thresholding.h`